### PR TITLE
DEPLOY-863: Consider Reverting DBP Deployment Requirements Definition on Master

### PR DESCRIPTION
**Pull request by cristina-sirbu**
Added documentation around how to upgrade DBP components

### CODE REVIEW

File: upgrade-dbp-components.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#Upgrade DBP Components
+
+The Digital Business Platform contains several different components, each with its own life cycle.
+To update any of them you should follow this process:
+1. Update the [requirements](https://github.com/Alfresco/alfresco-dbp-deployment/blob/master/helm/alfresco-dbp/requirements.yaml) file with the new version of the component.
+1. If you are updating DBP 1.5, please increase the version of the DBP chart in [Chart.yaml](https://github.com/Alfresco/alfresco-dbp-deployment/blob/develop/helm/alfresco-dbp/Chart.yaml#L2).
+1. Create a PR.
```
Comment (rgauss) [inline upgrade-dbp-components.md]: "Create a PR referencing a JIRA issue"

File: upgrade-dbp-components.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+1. The DBP test pipeline should be run when new versions of the chart are pushed to the Helm repository.
+1. You can talk to any member of the Platform Services team to review your PR.
```
Comment (rgauss) [inline upgrade-dbp-components.md]: I think I'd leave this line out.  We should already be aware of them from JIRA activity and GitHub notifications.

**Comment by cristina-sirbu** [comment]
@rgauss I've updated the docs with your suggestions in mind. Can you please take another look?

**Comment by cristina-sirbu** [comment]
@rgauss 
Done. :)

**Pull request merged**